containerd-shim-spin/src/engine.rs

```diff
@@ -40,10 +40,6 @@ const SPIN_HTTP_LISTEN_ADDR_ENV: &str = "SPIN_HTTP_LISTEN_ADDR";
 const RUNTIME_CONFIG_PATH: &str = "/runtime-config.toml";
 /// Describes an OCI layer with Wasm content
 const OCI_LAYER_MEDIA_TYPE_WASM: &str = "application/vnd.wasm.content.layer.v1+wasm";
```

Comment:
suggestion: make `spin_oci::client::WASM_LAYER_MEDIA_TYPE` public and use it in the Spin Shim.

Reply:
In fact, Spin itself is waiting on the canonical type/string upstream: https://github.com/fermyon/spin/blob/main/crates/oci/src/client.rs#L34-L35.  To be defined in https://github.com/bytecodealliance/registry perhaps?  Therefore expecting both Spin and the shim would pull this from the upstream crate.